apps/meteor/client/views/omnichannel/additionalForms/PrioritiesSelect.tsx

```
@@ -20,49 +17,26 @@ type PrioritiesSelectProps = {
-		(label: string, value: string) => {
-			return (
-				<>
-					<PriorityIcon level={sorting[value] || LivechatPriorityWeight.NOT_SPECIFIED} showUnprioritized /> {label}
-				</>
-			);
-		},
-		[sorting],
+	const formattedOptions = useMemo<SelectOption[]>(
+		() => [
+			['', t('Unprioritized')],
+			...options?.map(({ dirty, name, i18n, _id }) => [_id, dirty && name ? name : t(i18n as TranslationKey)] as SelectOption),
```

Biome (2.3.14)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)